tests/conftest.py

```diff
@@ -1,23 +1,50 @@
+from pathlib import Path
+
 import requests
 from pytest import fixture
 
 from snakesist import ExistClient
 
 
-ROOT_COLL = "/db/tests"
+@fixture
+def rest_base_url(test_client):
+    test_client.root_collection = "/db/tests"
+    return (
+        f"{test_client.base_url}rest{test_client.root_collection}?_wrap=no&_indent=no"
```

Comment:
considering that `base_url` doesn't end with a slash anymore, could it be that "rest" needs to have that slash prepended here?

Reply:
i get your impression, but prepending that slash actually results in a `//` in the final url and let's it fail. i haven't looked further into it b/c i anticipate further changes how `TestClient.base_url` is assembled w/ regards to #21.

Reply:
`…exist//rest…` to be precise.

Reply:
ok, that's good to know